extras/splat/splat-instance.js

```diff
@@ -32,11 +32,12 @@ class SplatInstance {
 
     lastCameraDirection = new Vec3();
 
-    constructor(splat, cameraEntity, debugRender = false) {
+    constructor(splat, options) {
         this.splat = splat;
 
         // material
-        this.material = splat.createMaterial(debugRender);
+        const debugRender = options.debugRender;
+        this.material = splat.createMaterial(options);
```

Comment:
It would be good if we also supported user passing in material.

Reply:
separate PR. But not sure how that would work with a compute shader implementation on WebGPU, so I would not do that yet.